### ISSUE TITLE
[Issue] Instant ID cuda Cannot copy out of meta tensor

### ISSUE THREAD

**Issue by Fearganainim** [vladmandic/automatic#3124]
### Issue Description

Instant ID throwing up an error on execution. Still persists after several clean reinstalls.
[sdnext.log](https://github.com/vladmandic/automatic/files/15297773/sdnext.log)

### Version Platform Description

OS Name	Microsoft Windows 11 Pro
Version	10.0.22631 Build 22631
Microsoft Edge
  Starting SD.Next
17:20:09-551555 INFO     Logger: file="E:\VLAD\vladmandic\sdnext.log" level=INFO size=52727 mode=append
17:20:09-552557 INFO     Python 3.10.11 on Windows
17:20:09-652401 INFO     Version: app=sd.next updated=2024-05-07 hash=e081f232 branch=master
                         url=https://github.com/vladmandic/automatic.git/tree/master
17:20:10-133516 INFO     Platform: arch=AMD64 cpu=AMD64 Family 25 Model 97 Stepping 2, AuthenticAMD system=Windows
                         release=Windows-10-10.0.22631-SP0 python=3.10.11
17:20:10-137516 INFO     nVidia CUDA toolkit detected: nvidia-smi present
17:20:10-179763 WARNING  Modified files: ['vlad.ico']
17:20:10-180763 INFO     Startup: standard
17:20:10-180763 INFO     Verifying requirements
17:20:10-189763 INFO     Verifying packages
17:20:10-190763 INFO     Verifying submodules
17:20:24-758336 INFO     Extensions enabled: ['Lora', 'sd-extension-chainner', 'sd-extension-system-info',
                         'sd-webui-agent-scheduler', 'sd-webui-controlnet', 'stable-diffusion-webui-images-browser',
                         'stable-diffusion-webui-rembg']
17:20:24-759336 INFO     Verifying requirements
17:20:24-767337 INFO     Updating Wiki
17:20:25-403519 INFO     Command line args: ['--upgrade'] upgrade=True
17:20:30-577195 INFO     Load packages: {'torch': '2.3.0+cu118', 'diffusers': '0.27.0', 'gradio': '3.43.2'}
17:20:31-635061 INFO     Engine: backend=Backend.DIFFUSERS compute=cuda device=cuda attention="Scaled-Dot-Product"
                         mode=no_grad
17:20:31-675575 INFO     Device: device=NVIDIA GeForce RTX 4090 n=1 arch=compute_37 cap=(8, 9) cuda=11.8 cudnn=8700
                         driver=552.22

### URL link of the extension

https://github.com/InstantID/InstantID

### Acknowledgements

- [X] I have read the above and searched for existing issues

**Comment by Fearganainim** [comment]
Logs Here:
[SD Next Log 21st April 2021.txt](https://github.com/vladmandic/automatic/files/15298483/SD.Next.Log.21st.April.2021.txt)

**Comment by vladmandic** [comment]
relevant log part:
```log
2024-05-13 17:11:25,404 | sd | DEBUG | instantid | InstantID loading: model=InstantX/InstantID
2024-05-13 17:11:25,503 | sd | DEBUG | sd_models | Setting model: enable VAE slicing
2024-05-13 17:11:28,888 | sd | ERROR | sd_models | Model move: device=cuda Cannot copy out of meta tensor; no data! Please use torch.nn.Module.to_empty() instead of torch.nn.Module.to() when moving module from meta to a different device.
2024-05-13 17:11:28,914 | sd | DEBUG | instantid | InstantID args: {'image_embeds': torch.Size([512]), 'image': <PIL.Image.Image image mode=RGB size=1138x1293 at 0x19D26541C60>, 'controlnet_conditioning_scale': 0.5, 'ip_adapter_scale': 1.0}
2024-05-13 17:11:28,943 | sd | ERROR | sd_models | Model move: device=cuda Cannot copy out of meta tensor; no data! Please use torch.nn.Module.to_empty() instead of torch.nn.Module.to() when moving module from meta to a different device.
2024-05-13 17:11:28,944 | sd | INFO | processing_diffusers | Base: class=StableDiffusionXLInstantIDPipeline
2024-05-13 17:11:28,950 | sd | DEBUG | processing_diffusers | Diffuser pipeline: StableDiffusionXLInstantIDPipeline task=DiffusersTaskType.TEXT_2_IMAGE set={'prompt': 10, 'negative_prompt': 0, 'guidance_scale': 6, 'generator': device(type='cuda'), 'num_inference_steps': 20, 'eta': 1.0, 'output_type': 'latent', 'width': 768, 'height': 1024, 'image_embeds': torch.Size([512]), 'image': <PIL.Image.Image image mode=RGB size=1138x1293 at 0x19D26541C60>, 'controlnet_conditioning_scale': 0.5, 'parser': 'Fixed attention'}
2024-05-13 17:11:28,969 | sd | ERROR | sd_models | Model move: device=cuda Cannot copy out of meta tensor; no data! Please use torch.nn.Module.to_empty() instead of torch.nn.Module.to() when moving module from meta to a different device.
```

**Comment by Fearganainim** [comment]
It's strange, the extension worked correctly prior to the last update. Thanks. Fantastic Repo by the way.

**Comment by Fearganainim** [comment]
Working again now after update. I suspect something to do with diffusers. Anyhoo all good.
